CHALLENGE 2.0/unique_string.py

```diff
@@ -0,0 +1,5 @@
+def remove_duplicates(string_list):
+    return set(string_list)
+
+
+print(remove_duplicates('namedominionjohneleojo'))
```

Comment:
This returns `{'m', 'h', 'a', 'i', 'l', 'o', 'j', 'e', 'd', 'n'}`. It should return a string like "adelhijmno"